airbase-policy/src/main/resources/checkstyle/airbase-checks.xml

```diff
@@ -49,6 +49,10 @@
         <property name="format" value="^import static .*\.(of|copyOf|valueOf);$" />
         <property name="message" value="The following methods may not be statically imported: of, copyOf, valueOf" />
     </module>
+    <module name="RegexpSingleline">
+        <property name="format" value="^import static (?!java\.lang\.String\.format;).*\.format;" />
```

Comment:
Should the pattern end in `$`? We do that for the others

Reply:
Why would i want it to end with `$`?